src/presets/cloudflare.ts

```diff
@@ -69,17 +68,6 @@ const cloudflarePreset: Preset = {
     process: "unenv/runtime/node/process/$cloudflare",
     setImmediate: ["unenv/runtime/node/timers/$cloudflare", "setImmediate"],
     clearImmediate: ["unenv/runtime/node/timers/$cloudflare", "clearImmediate"],
-    performance: ["perf_hooks", "performance"],
-    Performance: ["perf_hooks", "Performance"],
-    PerformanceEntry: ["perf_hooks", "PerformanceEntry"],
-    PerformanceMark: ["perf_hooks", "PerformanceMark"],
-    PerformanceMeasure: ["perf_hooks", "PerformanceMeasure"],
-    PerformanceObserver: ["perf_hooks", "PerformanceObserver"],
-    PerformanceObserverEntryList: [
-      "perf_hooks",
-      "PerformanceObserverEntryList",
-    ],
-    PerformanceResourceTiming: ["perf_hooks", "PerformanceResourceTiming"],
```

Comment:
Since the cloudflare variant is only broken, I think disabling these would be enough (and hybrid from L32) should be enough?

Happy to reiterate in another PR if it is not